koku/api/report/aws/aws_query_handler.py

```diff
@@ -64,9 +64,10 @@ def annotations(self):
             (Dict): query annotations dictionary
 
         """
+        units_fallback = self._mapper._report_type_map.get('units_fallback')
         annotations = {
             'date': self.date_trunc('usage_start'),
-            'units': Concat(self._mapper.units_key, Value(''))
+            'units': Coalesce(Value(units_fallback), Concat(self._mapper.units_key, Value('')))
```

Comment:
coalesce will return the first non-null arg, so units_fallback will always get returned.

Reply:
yep, I forgot to flip this one. Will fix. Nice catch.